### ISSUE TITLE
NFR is exclusionary

### ISSUE THREAD

**Issue by kxrt** [kxrt/pe#17]
**Description**

NFR 11 states "BookFace does not plan to improve its accessibility for blind, deaf, or otherwise disabled users
according to the Web Content Accessibility Guidelines (WCAG 2.1).", which may be considered exclusionary against users who suffer from these a11y issues. Furthermore, the WCAG is for *web* applications and not *desktop* applications, and cannot apply here.
